README.md

```diff
@@ -61,12 +55,13 @@ Then navigate to `http://127.0.0.1:8000` in your browser.
 
 * Cross-platform using PowerShell Core (with support for PS5)
 * Docker support, including images for ARM/Raspberry Pi
+* Azure Functions, AWS Lambda, and IIS support
 * OpenAPI specification version 3.0.x and 3.1.0
 * OpenAPI documentation with Swagger, Redoc, RapidDoc, StopLight, OpenAPI-Explorer and RapiPdf
-* OpenAPI, Swagger, and ReDoc support
 * Listen on a single or multiple IP(v4/v6) address/hostnames
 * Cross-platform support for HTTP(S), WS(S), SSE, SMTP(S), and TCP(S)
 * Host REST APIs, Web Pages, and Static Content (with caching)
+* Async REST APIs with optional callback
```

Comment:
This one looks to have leaked in from #1349 😉

Reply:
I can remove from here 😏